src/components/Map.tsx

```diff
@@ -22,9 +22,9 @@ function Map() {
        * Initialize application
        */
 
-      const globalApikey =
-        "AAPK66d0fa7f737142fcb780f9fd1c8155f04YBLPof_SmYB-dzJgafylrodNieETG6SVXdYeo75C5eFIt9NykPed6jRtTI_c01w";
-      esriConfig.apiKey = globalApikey;
+      const { REACT_APP_GLOBAL_API_KEY } = process.env;
+
```

Comment:
Nitpick: Can we be more specific just in case we need to add additional API keys, e.g. `REACT_APP_GLOBAL_ARCGIS_API_KEY`. Would Survey123 need its own key?

Reply:
There is only ever one global API key. I can be more specific with the other API keys when needed - e.g. for spatial analysis services, I can use `REACT_APP_SPATIAL_ANALYSIS_API_KEY`. All the the others will need to be fine-grained or be added to the global API key.